bootstrap.py

```diff
@@ -288,11 +301,24 @@ def install_puppet_agent():
     exec_failexit("/sbin/service puppet restart")
 
 
+def install_pe_agent(org_name):
+    org_label = return_matching_katello_key('organizations', 'name="%s"' % org_name, 'label', False)
+    if os.path.isfile('/etc/yum.repos.d/pe_repo.repo'):
+        exec_failexit("mv /etc/yum.repos.d/pe_repo.repo{,.bak}")
+    exec_failexit("subscription-manager attach --pool=$(subscription-manager list --available --matches='Puppet Enterprise' --pool-only)" )
+    exec_failexit("subscription-manager repos --disable=*-pe-*")
+    exec_failexit("subscription-manager repos --enable=%s_Puppet_Enterprise_$(uname -r | cut -d. -f6)-pe-$(uname -r | cut -d. -f7)" % ( org_label ))
```

Comment:
those two should be done using `--enablerepos` as we don't know how the repos will be called at the users setup